doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2442,7 +2442,13 @@ nRF5340 Audio
 
 The issues in this section are related to the :ref:`nrf53_audio_app` application.
 
-.. rst-class:: v3-1-0
+.. rst-class:: v3-1-0 v3-1-1
+
+OCT-3265: Test with nRF5340DK as sink will cause the I2S signals disrupt signals on the same pins.
+  Using the nRF5340DK to drive an external digital-to-analgue converter via I2S will not produce audio.
```

Comment:
```suggestion
  Using the nRF5340DK to drive an external digital-to-analog converter via I2S will not produce audio.
```

Reply:
Sorry we use American English (not that I can spell in English English!)